posthog/models/filters/test/test_filter.py

```diff
@@ -325,6 +326,8 @@ def test_filter_out_team_members_persons(self):
 
 
 def _filter_persons(filter: Filter, team: Team):
+    # TODO: confirm what to do here?
+    # Postgres only supports ANDing all properties :shrug:
```

Comment:
For this TODO, I think it's okay to not support ORs on postgres. So just go with flat. Agreed?